src/components/comment/comment.scss

```diff
@@ -83,6 +98,24 @@
   margin-top: math.div(size.$rhythm-condensed, -4);
 }
 
+// We added default styles to the blockquote element using the `wp-block-quote`
+// class. We're applying those same styles here, so comment blockquotes look
+// the similar.
+
+.c-comment__content blockquote {
+  border-left: size.$border-width-blockquote solid
+    var(--theme-color-border-blockquote);
+  color: var(--theme-color-text-blockquote);
+}
+
+.c-comment__content blockquote cite {
+  font-style: normal;
+}
+
+.c-comment__content blockquote cite::before {
+  @include emdash.content;
```

Comment:
Just curious if we could simplify by using an em dash directly instead of needing the `\` code?

```
content: '—';
```